packages/openactive-integration-tests/test/features/leasing/common.js

```diff
@@ -0,0 +1,150 @@
+const { expect } = require('chai');
```

Comment:
most of the functions in here were duplicated in `anonymous-leasing/implemented/lease-opportunity-capacity-update-test.js` and `named-leasing/implemented/lease-opportunity-capacity-update-test.js`